vm/devices/storage/disk_nvme/nvme_driver/src/tests.rs

```diff
@@ -208,7 +208,7 @@ async fn test_nvme_driver(driver: DefaultDriver, allow_dma: bool) {
         .await
         .unwrap();
 
-    assert_eq!(driver.fallback_cpu_count(), 0);
+    assert_eq!(driver.fallback_cpu_count(), 2);
```

Comment:
This is a bug: there should not be any fallback CPUs if the number of IO issuers is less than the number of CPUs.